events/race.md

```diff
@@ -56,11 +56,35 @@ Indicates that racers should start racing.
 
 Indicates the racer has passed a gate, possibly completed a lap, 
 
-- TODO: Rename this or break it into multiple events? `racer_completed_lap` and `racer_passed_gate`
-- TODO: This doesn't indicate the gate, should it?
+### Properties
+- `fast` [boolean]: Racer's fastest lap
+- `streak` [boolean]: Indicates if the racer is on a streak
+- `valid` [boolean]: Valid gate pass
+- `transponder` [string]: The transponder ID of the racer
+- `gate` [string]: The id of the gate the racer passed
+- `gate_type` [string start|checkpoint|finish]: The type of gate the racer passed
 
 ```json
-{"cmd":"event","evt":"race","type":"racer_passed_gate","fast":false,"streak_laps":0,"lap":true,"transponder":"123",[...protocol properties]}
+{"cmd":"event","evt":"race","type":"racer_passed_gate","fast":false,"streak":true,"lap":true,"transponder":"123","gate_id":"1","gate_type":"start",[...protocol properties]}
+```
+
+
+## Race Standings
+`Occurs When`: An update to the standings has occurred.
+
+The position of the racers must be calculated based on the race type.
+
+### Properties
+- `name` [string]: Racer's name
+- `laps` [integer]: Number of laps completed
+- `fast` [integer]: Fastest lap time (in milliseconds)
```

Comment:
I'd use a different field name then what you have above to avoid confusion. Maybe "fastTime", or "fastest"

Reply:
Updated this to be

- `fast_lap` [integer]: Racer's fastest lap time of the race (in milliseconds)